### ISSUE TITLE
Alpine aarch64/arm64 support

### ISSUE THREAD

**Issue by willat8** [confluentinc/confluent-kafka-dotnet#2341]
I notice `librdkafka.redist` provides Linux arm64 `librdkafka.so` but not `alpine-librdkafka.so`. It would be great to get the same for Alpine.

**Comment by grietine** [comment]
> I notice `librdkafka.redist` provides Linux arm64 `librdkafka.so` but not `alpine-librdkafka.so`. It would be great to get the same for Alpine.

I agree. It would be valuable if this NuGet would support Alpine Arm64.


We are using Alpine image, but with amd64 architecture:
`mcr.microsoft.com/dotnet/aspnet:8.0-alpine`

We want to migrate to arm64 base architecture and continue using the Alpine image, but I noticed that this NuGet stopped working. Error:
`System.DllNotFoundException: Failed to load the librdkafka native library`

After investigating I found that there **are missing dependencies** for `librdkafka.so` **in Alpine as it uses musl instead of glibc**:
```
/app/runtimes/linux-arm64/native # ldd librdkafka.so
        /lib/ld-musl-aarch64.so.1 (0xffff89818000)
        libm.so.6 => /lib/ld-musl-aarch64.so.1 (0xffff89818000)
        libdl.so.2 => /lib/ld-musl-aarch64.so.1 (0xffff89818000)
        libpthread.so.0 => /lib/ld-musl-aarch64.so.1 (0xffff89818000)
        librt.so.1 => /lib/ld-musl-aarch64.so.1 (0xffff89818000)
        libc.so.6 => /lib/ld-musl-aarch64.so.1 (0xffff89818000)
Error relocating librdkafka.so: getcontext: symbol not found
Error relocating librdkafka.so: __getauxval: symbol not found
Error relocating librdkafka.so: makecontext: symbol not found
Error relocating librdkafka.so: setcontext: symbol not found
```

**Switching back to the default Debian based (which has the missing packages) it works on Arm64:**
`mcr.microsoft.com/dotnet/aspnet:8.0`

Alternative way would be to compile the **librdkafka.redist** by ourselves, but this doesn't seem like a viable solution in the long term. 

Is there a plan to support Alpine with Arm64?

**Comment by willat8** [comment]
A workaround I am using is to install the librdkafka alpine package from edge and symlink it to the expected location. Brittle but works for now.

**Comment by ChaitanyaTummala7** [comment]
I am also facing similar issue

**Comment by vgmello** [comment]
+ 1

**Comment by jaironalves** [comment]
+ 1
